types/paypal-rest-sdk/index.d.ts

```diff
@@ -0,0 +1,761 @@
+// Type definitions for paypal-rest-sdk 1.7
+// Project: https://github.com/paypal/PayPal-node-SDK
+// Definitions by: My Self <https://github.com/trainerbill>
```

Comment:
`My Self` is an odd thing to be credited as; just sayin'. Just put your github username if a real name is too much to ask.